data.sql

```diff
@@ -13,4 +13,64 @@ VALUES ('Charmander', '2020-02-08', 0, false, -11 ),
 ('Angemon', '2005-06-12', 1, true, -45 ),
 ('Boarmon', '2005-06-07', 7, true, 20.4 ),
 ('Blossom', '1998-10-13', 3, true, 17 ),
-('Ditto', '2022-05-14', 4, true, 22 );
\ No newline at end of file
+('Ditto', '2022-05-14', 4, true, 22 );
+
+INSERT INTO owners(full_name,age )
+VALUES('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name )
+VALUES('Pokemon'),
+('Digimon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name ILIKE '%mon%';
+
+UPDATE animals
+SET species_id = 1
+WHERE NOT name ILIKE '%mon%';
+
+BEGIN;
+UPDATE animals
+SET owner_id = 1
+WHERE name = 'Agumon';
+
```

Comment:
* ## As stated in the requirement you are to modify your inserted data here which are missing kindly add all modifications here 
![image](https://user-images.githubusercontent.com/52497246/236370965-7d945d7b-a892-4f65-b63b-2d7e25173554.png)